recipes/python-bioext/build.sh

```diff
@@ -9,3 +9,15 @@ fi
 export FREETYPE2_ROOT=$PREFIX
 
 $PYTHON setup.py install --single-version-externally-managed --record=record.txt
+
+for CHANGE in "activate" "deactivate"
+do
+    mkdir -p "${PREFIX}/etc/conda/${CHANGE}.d"
+    cp "${RECIPE_DIR}/${CHANGE}.sh" "${PREFIX}/etc/conda/${CHANGE}.d/${PKG_NAME}_${CHANGE}.sh"
+done
+
+# Symlink reference data to /usr/local/share, since the site-packages path may vary depending on python version
+mkdir -p "${PREFIX}/usr/local/share/bealign_references"
+ln -s "$SP_DIR/BioExt/data/references/hxb2" "$PREFIX/usr/local/share/bealign_references/HXB2"
```

Comment:
Instead of specifying each reference individually you could do something like:

`ln -s "$SP_DIR/BioExt/data/references" "$PREFIX/usr/local/share/bealign_references"`

Though looking at the repo this was the first time they added a reference in more than 9 years.